code/src/Eduweb/TrainingBundle/Resources/views/Blog/Widgets/followingWidget.html.twig

```diff
@@ -0,0 +1,13 @@
+
+{% import 'EduwebTrainingBundle:Blog:macros.html.twig' as blogMacros %}
+{{ blogMacros.widgetList('Aktualnie oberwuję',list,'Wszystkie spółki',"http://www.gpw.pl/lista_spolek") }}
+
+{#<section class="widget">#}
+{#    <h2>Aktualnie oberwuję</h2>#}
+{#    <ul>#}
+{#        {% for item in list %}#}
+{#            <li>{{  item }}</li>#}
+{#        {% endfor %}#}
+{#    </ul>#}
+{#    <p><a class="btn btn-default" href="http://www.gpw.pl/lista_spolek" role="button">Wszystkie spółki &raquo;</a></p>#}
+{#</section>#}
```

Comment:
Why is this code is commented out? We should avoid to commented out code and pushing it to GH.